library/pod-security-policy/host-filesystem/template.yaml

```diff
@@ -16,13 +18,16 @@ spec:
           properties:
             allowedHostPaths:
               type: array
+              description: "An array of hostpath objects, representing paths and read/write configuration."
               items:
                 type: object
                 properties:
-                  readOnly:
-                    type: boolean
                   pathPrefix:
                     type: string
+                    description: "The path prefix that the host volume must match."
```

Comment:
Rather than invoking K8s resource specifics (which we are attempting to insulate users from), could we change this to:

"The prefix that the host path backing the host volume must match"

In general: prefer intent-based descriptions vs. schema-based descriptions, b/c the schema may change across resource versions while the intent stays the same

Reply:
Then what about a link to the schema? It can be a pain to find an example of the schema we're referencing.

Reply:
That sounds like it would be covered by adding "examples" documentation that shows example allowed/disallowed resources, which we can source from test data?